sympy/stats/crv_types.py

```diff
@@ -2770,6 +2825,19 @@ def StudentT(name, nu):
 class TrapezoidalDistribution(SingleContinuousDistribution):
     _argnames = ('a', 'b', 'c', 'd')
 
+    @property
+    def set(self):
+        return Interval(self.a, self.d)
+
+    @staticmethod
+    def check(a, b, c, d):
+        _value_check(a < d, "Lower bound parameter a < %s. a = %s"%(d, a))
+        _value_check(And(a <= b, b < c),
```

Comment:
According to latest refactoring of `_value_check` ,`And(a <= b, b < c)` should be written as, `(a <= b, b < c)`.

Reply:
> According to latest refactoring of `_value_check` ,`And(a <= b, b < c)` should be written as, `(a <= b, b < c)`.

+1

Reply:
Done. Thanks!